builtin/providers/librato/common.go

```diff
@@ -0,0 +1,40 @@
+package librato
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Encodes a hash into a JSON string
+func attributesFlatten(attrs map[string]string) (string, error) {
```

Comment:
I've deleted this file for now. There is a decent amount of duplication within the Librato plugin (as well as the AWS plugin) that needs to be rectified, but that's a task for another PR. :-) . If I need these though, I'll use them vs the Librato specific ones so as to get us started with removing them.